oslabs-beta/ApolloDevQL
# Bug: Similar Dev Tool messaging conflict & QL Operations with empty operation names

Feature:
Fixed Bug associated with presence of Apollo Client Dev Tool causing conflicts in message plumbing bwt ApolloDevQL's hook and dev panel.
Fixed Bug associated with Empty Operation names (hoping to avoid naming events as Mutation or Query)

Description
-     Added queries & mutations key to EventLogStore type.
-     Added a helper method to validate operation.name.value and return a valid operationName as against undefined when apollo client QLs have no operation name in the root document/mutation object
-     Now included root queries & mutations from the hook callback as an alternate source of truth to EventObject.sequenceApolloLog to help fetch keys that are not in the constructed QueryManager.

How to Test
1. npm run build
2. open this website https://shopping.qantas.com/apple, perform site operations
3. Observe the ApolloDevQL panel.
